### ISSUE TITLE
Add API Scripts

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/Awesome_Python_Scripts#22]
This issue is specifically created for those who have basic programming knowledge in python and want to explore on API scripts on python

Add basic and advance API scripts in 'APIScripts' folder.

Repository : APIScripts
Level : Easy and Medium

Do check if the code you want to contribute is available in this repository
Comment on this issue with the title and details of the script you are going to contribute and what does it do
Start working on that script and before you add make sure it works properly
Add your code to 'APIScripts' repository using the contribution guidelines and template mentioned
Create a Pull Request and be patient
We will review your code and if all works perfectly fine, PR will be merged.
If anything else needed from you, we will get back with the response
Make good PR and clean code.
Any queries, drop me a private message or just mention me over here!. I'll be happy to support and mentor you.
Best of luck everyone ❤️

Happy Contribution!

**Comment by mkhmirza** [comment]
Hello, I going to contribute a python api for [Solar System Open Data](https://api.le-systeme-solaire.net/). This api displays our solar system's planet information.

**Comment by prathimacode-hub** [comment]
Wow, sounds interesting. Go ahead. Kindly follow the template mentioned in contributing guidelines to avoid merge conflicts. All the best @mkhmirza

**Comment by RandallWert** [comment]
Hello, I have a couple of APIs I could contribute. They're already in [a GitHub repo of my own](https://github.com/RandallWert/Fun_with_APIs), so I don't know if you still want them? If so, I would be happy to contribute them. My two favorites are one that lists earthquakes in Oregon and Pennsylvania (two states where my family lives) in the past week and another that queries a thesaurus; this one features a simple GUI.

**Comment by prathimacode-hub** [comment]
Oh, that's great. Sounds interesting. Yeah, you are welcome to contribute. Being in a repo of your own, doesn't matter. This repo is created to encourage open source. Looking forward to your contributions. @RandallWert

**Comment by RandallWert** [comment]
OK, great. But I am a little confused about this line: "Add your code to 'APIScripts' repository using the contribution guidelines and template mentioned." How exactly do I add to the repository? And where is the template?

Sorry for these beginner questions.

**Comment by prathimacode-hub** [comment]
'APIScripts' is a folder name inside Awesome_Python_Scripts. And the project you are contributing should be in that concerned folder. 
To get your understanding better : Your project should be in 
"https://github.com/prathimacode-hub/Awesome_Python_Scripts/APIScripts" repository.

Template is given inside [CONTRIBUTING_GUIDELINES.md](https://github.com/prathimacode-hub/Awesome_Python_Scripts/blob/main/CONTRIBUTING_GUIDELINES.md)
This will help you get the information of how file names and folder name should look like. 
(For Ex: If hello world program is added. Folder name should be "HelloWorld" and "Hello_World.py"

README.md should be included as well along with program file. 
README should be showing up clearing understanding of what project is about, how it works etc.
Any requirements or libaries needed it can be added up in Requirements.txt file.
Also add up your output screenshot in README.md

Nope, it's good you had asked. Hope this cleared your doubts. Anymore queries, feel free to ask.
Good to see your initiation to contribute. @RandallWert

**Comment by RandallWert** [comment]
Thanks so much for the detailed answer and encouragement! I'll give it a try.

**Comment by prathimacode-hub** [comment]
Sure, go ahead @RandallWert

**Comment by RandallWert** [comment]
Is it possible to create the README.md directly on GitHub?

**Comment by prathimacode-hub** [comment]
Yes, it is possible. While creating a file, all you have to is write README.md as filename and it's automatically created. @RandallWert

**Comment by RandallWert** [comment]
OK, I'm still confused. Do I create the folder on GitHub somehow, or on my own machine? I haven't been able to create it on GitHub. I can upload individual files, but I can't upload an entire folder.

Putting it differently, I got as far as creating a branch, but I am having trouble committing my new folder to that branch.

**Comment by prathimacode-hub** [comment]
You upload individual files in a folder, say it a file_name.py
To create a folder, Go to Add File Option. In that select, Create a New File 
Now a blank page is open. At the space of writing your file name, give your folder name and add a / 
Then you can add a blank README.md file so that the folder name given above is set.
Finally, you can add your relevant program script .py file and fill up your README.md file created earlier.

Hope, it's clear. @RandallWert

**Comment by RandallWert** [comment]
OK, now it _might_ have worked. I'll try to do the pull request next.

**Comment by prathimacode-hub** [comment]
Your PR is merged successfully. Congrats. 🎉@RandallWert

**Comment by RandallWert** [comment]
Yay!!! :-)
